modules/ossm-about-concepts-resources.adoc

```diff
@@ -0,0 +1,113 @@
+
+[NOTE]
+====
+* Though {SMProductName} APIs are built around Istio's Helm chart APIs, Helm charts are not supported.
+* All installation and configuration options that are exposed by Istio's Helm charts are available through the {SMProductName} Custom Resource Definition (CRD) `values` fields.
+====
+
+[id="istio-resource_{context}"]
+== Istio resource
+
+The `Istio` resource is used to manage your Istio control planes. It is a cluster-wide resource, as the Istio control plane operates in and requires access to the entire cluster.
+
+To select a `namespace` to run the control plane pods in, you can use the `spec.namespace` field.
+
+[NOTE]
+====
+The `spec.namespace` field is immutable: in order to move a control plane to another namespace, you have to remove the `Istio` resource and recreate it with a different `spec.namespace`.
+====
+
+You can access all `Istio` custom resource definition (CRD) options through `spec.values` fields:
+
+.Example `Istio` resource CRD
+[source,yaml]
+----
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.22.3
+  namespace: istio-system
+  updateStrategy:
+    type: InPlace
+  values:
+    pilot:
+      resources:
+        requests:
+          cpu: 100m
+          memory: 1024Mi
+----
+
+You can run the following command to see all the customization options:
+
+[source, terminal]
+----
+$ oc explain istios.spec.values
+----
+//Helm charts can only be used with istio-csr and Helm charts are a temporary work around for istio-csr. This might get confusing for users, so a NOTE was added.
+
+To support canary updates of the control plane, {SMProduct} includes support for multiple Istio versions. You can select a version by setting  `spec.version` to the version you would like to install, prefixed with a v. You can update to a new version just by changing this field.
+
+{SMProduct} supports two different update strategies for your control planes:
+
+`InPlace`:: {SMProduct} Operator immediately replaces your existing control plane resources with the ones for the new version.
+
+`RevisionBased`:: Uses Istio's canary update mechanism by creating a second control plane to which you can migrate your workloads to complete the update.
+
+After creating an Istio resource, {SMProduct} generates a revision name for the resource based on the `updateStrategy`, and creates a corresponding `IstioRevision`.
+
+[id="istiorevision-resource_{context}"]
+== IstioRevision resource
+
+The `IstioRevision` is a cluster-wide resource and the lowest-level API {SMProduct} provides. It is usually not created by the user, but by the operator itself. Its schema closely resembles that of the `Istio` resource - but instead of representing the state of a control plane you want to be present in your cluster, it represents a revision of that control plane.
```

Comment:
```suggestion
The `IstioRevision` is a cluster-wide resource and the lowest-level API {SMProduct} provides. It is usually not created by the user, but by the Operator itself. Its schema closely resembles that of the `Istio` resource - but instead of representing the state of a control plane you want to be present in your cluster, it represents a revision of that control plane.
```

Do we need any mention of when/why the resource wouldn't be created by the Operator?

Reply:
Not for TP1.